components/director/internal/domain/formation/service_test.go

```diff
@@ -247,6 +247,14 @@ func TestServiceCreateFormation(t *testing.T) {
 		State:               model.ReadyFormationState,
 	}
 
+	expectedFormationInInitialState := &model.Formation{
```

Comment:
**_minor:_** Could we use a fixture here - `expectedFormationInInitialState := fixFormationModelWithState(model.InitialFormationState)`. ?